pkg/util/meta/meta.go

```diff
@@ -18,7 +18,7 @@ This file may have been modified by The KubeAdmiral Authors
 are Copyright 2023 The KubeAdmiral Authors.
 */
 
-package util
```

Comment:
`DeepCopyRelevantMeta` seems to be unused. Should we remove it?

Reply:
Along with many other functions in this package: `ObjectMetaEquivalent`, `ObjectMetaAndSpecEquivalent`, `MetaAccessor`, `GetUnstructured`, `GetPluralName`.

Reply:
Removed